client/my-sites/plans/current-plan/jetpack-checklist/constants.ts

```diff
@@ -19,7 +19,7 @@ interface TaskUiDescription {
 	readonly description?: string;
 	readonly completedButtonText: string;
 	readonly completedTitle?: string;
-	readonly getUrl: ( siteSlug: SiteSlug ) => string;
+	readonly getUrl: ( siteSlug: SiteSlug, completed?: boolean ) => string;
```

Comment:
I actually love how the types can help us relax our naming conventions to things that flow more naturally. normally I'd suggest we use `isCompleted` simply to indicate that it's a boolean value but in this case we know on hover that it's boolean. at some level the naming is still relevant because maybe there's another possible boolean value which `completed` could be holding, but I like this use here.

note too that we have `T.URL` if indeed this is supposed to return a URL

```ts
import { * as T } from 'types';

T.URL
```